magik-lint/src/main/java/nl/ramsolutions/sw/magik/lint/MagikLint.java

```diff
@@ -106,14 +107,51 @@ void showChecks(final Writer writer, final boolean showDisabled)
         continue;
       }
 
-      for (final MagikCheckHolder.Parameter parameter : holder.getParameters()) {
+      showParameters(writer, holder, showDefaultValues);
+    }
+  }
+
+  /**
+   * Show parameters for a check.
+   *
+   * @param writer Writer Write to write output to.
+   * @param showDefaultValues boolean Boolean to show default values for parameters or not.
```

Comment:
No need to state that it is a boolean. Java includes types for their parameters etc. :)